crates/api/src/site/purge/community.rs

```diff
@@ -46,16 +45,6 @@ pub async fn purge_community(
   )
   .await?;
 
-  if let Some(banner) = &community.banner {
-    purge_image_from_pictrs(banner, &context).await.ok();
-  }
-
-  if let Some(icon) = &community.icon {
-    purge_image_from_pictrs(icon, &context).await.ok();
-  }
```

Comment:
For 1.0 it is not possible to set an image url now, but only to upload the image data directly. So its is fine to call `delete_image_from_pictrs()` for these.

Reply:
it might still come in via federation. this does not check that the community is local.
additionally, even if deletion is desired, it should be a delete call to pictrs, not a purge call.

Reply:
for users, this is generally covered by the call to delete all media uploaded by the user, so this doesn't need special handling for avatar and banner.

it does however remove this from bans. i already mentioned this in #5560, it doesn't seem like a good idea to include any media deletion in user bans, only in user purges, as this is not a reversible action.

Reply:
This makes sense, since those images might have been uploaded by an innocent person.